Tests/test_numpy.py

```diff
@@ -12,6 +12,7 @@
 
 
 class TestNumpy(PillowTestCase):
+    TEST_IMAGE_SIZE = (10, 10)
```

Comment:
This should probably be a module level global, rather than a class member. 

So all of the TestNumpy.TEST_IMAGE_SIZE can be just TEST_IMAGE_SIZE